Sources/JExtractSwift/TranslatedType.swift

```diff
@@ -274,6 +294,15 @@ enum CCompatibleJavaMemoryLayout: Hashable {
   case cFunction
 }
 
+enum SwiftTypeKind {
```

Comment:
TODO: Working on remove this and converge with the other `SwiftNominalTypeDeclaration.Kind` in a follow up PR